ducktape/tests/reporter.py

```diff
@@ -226,7 +226,7 @@ def report(self):
                     name=name, classname=test.cls_name, time=str(test.run_time_seconds),
                     status=str(test.test_status), assertions=""
                 ))
-                if test.test_status == FAIL:
+                if test.test_status == FAIL or test.test_status == OFAIL:
                     xml_failure = ET.SubElement(xml_testcase, 'failure', attrib=dict(
```

Comment:
maybe @andrewhsu is more suitable to answer and review, but shouldn't it contain `'ofailure'` and not `'failure'` in `OFAIL`ed cases?

Reply:
I'm sort of squeezing this to fit within the junit XML format: he 'failure' element is the junit format's place for putting failure reasons.  That format doesn't have a concept of ofail, so it feels neater to use the existing element than invent a new one.

Worst case, this would confuse other junit-compatible tooling into thinking that ofailures are really failures, but the alternative of inventing a new element would probably also confuse other tools.